src/atomate2/abinit/utils/settings.py

```diff
@@ -0,0 +1,45 @@
+"""Settings for abinit flows in atomate2."""
+
+from abipy.flowtk.tasks import TaskManager
+from pydantic import BaseSettings, Field
+
+
+class AbinitAtomateSettings(BaseSettings):
```

Comment:
This could be supported but I'm worried it would be confusing for users. With pydantic, you can override settings using environment variables. E.g., you can set the following in your job file or .bashrc:
```bash
export ATOMATE2_VASP_CMD="vasp_std"
```

However, if using a nested approach, I think this would have to be written with a double `__` between VASP and the nested option. [See here for more details.](https://pydantic-docs.helpmanual.io/usage/settings/#parsing-environment-variable-values) E.g.
```bash
export ATOMATE2_VASP__CMD="vasp_std"
```

I can see lots of people making this mistake. For that reason, I think its better to have all options on the root level. And practically in python it doesn't make much difference between `SETTINGS.VASP_CMD` and `SETTINGS.VASP.CMD`.

Happy to discuss further if this is something you feel strongly about.

Reply:
Thanks for pointing this out. I did not know about this option of the double underscore for pydantic. Indeed, this could easily become a common mistake.  
I thought splitting the options could help keeping the configuration file in order, since I imagine that at some point many codes will be present, each with their own options. However, I do not have a strong feeling about this.